doc/user_guide/transformations.rst

```diff
@@ -128,11 +128,12 @@ The generic transformations currently available are listed in
 alphabetical order below (a number of these have specialisations which
 can be found in the API-specific sections).
 
-.. note:: PSyclone currently only supports OpenCL and KernelGlobalsToArguments
-          transformations for the GOcean 1.0 API, the OpenACC Data
-          transformation is limited to the NEMO and GOcean 1.0 APIs and the
-          OpenACC Kernels transformation is limited to the NEMO and Dynamo0.3
-          APIs.
+.. note:: PSyclone currently only supports OpenCL and
```

Comment:
I take your point but these are meant to be temporary restrictions on what should be generic transformations (otherwise they should be in the relavant API-specific sections), so hopefully the restrictions will gradually disappear, rather than grow.

Reply:
OK, I'll let it go for the moment...